interface-definitions/vpp.xml.in

```diff
@@ -833,6 +833,51 @@
         </node>
       </children>
     </node>
+    <node name="nat">
```

Comment:
I wonder if we should use `nat44` name here. There are loads of NAT plugins for VPP already.

Reply:
@dmbaturin, which option is preferable?
```
set vpp nat nat44 source
```
or 
```
set vpp nat44 source
```

Reply:
renamed `set vpp nat` to `set vpp nat44`